src/services/polkadot-pallets/marketplaceApi.js

```diff
@@ -1,10 +1,13 @@
 import BasePolkadotApi from '~/services/basePolkadotApi'
+import { Rbac } from '../polkadot-pallets'
 // import markets from '~/services/const/allMarkets'
 // import participants from '~/services/const/participants'
 // import applicants from '~/services/const/applicants'
 class MarketplaceApi extends BasePolkadotApi {
   constructor (polkadotApi, notify) {
     super(polkadotApi, 'gatedMarketplace', notify)
+    this.RBAC = new Rbac(polkadotApi)
+    this.palletId = '0x5d208df71427902157988a56e7527474a7dce921f9d5efa159dcfc849289a53c'
```

Comment:
how we can get this dynamically?

Reply:
Yes, I think so, there is a way to get the pallet ID. I'll work on it. Thanks!